packages/block-editor/src/components/gradient-picker/panel.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { isEmpty } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { PanelBody } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import GradientPicker from './control';
+
+export default function GradientPanel( props ) {
```

Comment:
It's a bummer that we need to have these overrides for colors, gradients... I'd love if this is taken care of without having to introduce new APIs each time. I think we can probably make it non-experimental since it's the same as the colors one.

And maybe with the hoosk API `useColors` all these can be absorbed in a unique API.

Reply:
It is not the same as the color one, as the color panel one deals with multiple colors. I think it probably makes sense to have only one panel for gradients and colors. That's the reason why I kept this experimental for now.

Reply:
ok 👍 makes sense